### PR TITLE
New Crashes UI

### PR DESCRIPTION
**Pull request by LVMBDV**
It's not finished but take a look anyways 🥺

### CODE REVIEW

File: plugins/crashes/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "countly-crashes",
-  "title": "Crash analytics",
+  "title": "Crash analytics (Vue)",
```
Comment (can-keklik) [inline plugins/crashes/package.json:3]: Well, why `(Vue)`?

File: plugins/crashes/frontend/public/templates/overview.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                <template v-slot:header-left>
+                    <h2>
+                        Crash Statistics
+                        <cly-tooltip-icon></cly-tooltip-icon>
+                    </h2>
+                </template>
+            </cly-header>
+            <cly-main>
+                <cly-section class="crashes-statistics-dashboard">
+                    <div class="bu-columns bu-is-gapless bu-is-mobile" style="height: 314px;">
```
Comment (can-keklik) [inline plugins/crashes/frontend/public/templates/overview.html:46]: Inline styling can be refactored :)

File: plugins/crashes/frontend/public/javascripts/countly.models.js [hunk 3]
```diff
@@ -1,1142 +1,726 @@
...
+                var chartOptions = {
+                    xAxis: {
+                        data: results.chartData.map(function(dp) {
+                            return dp.date;
+                        })
+                    },
+                    series: []
+                };
+
+                [0, 1].forEach(function(index) {
```
Comment (can-keklik) [inline plugins/crashes/frontend/public/javascripts/countly.models.js:208]: noice :)

File: plugins/crashes/frontend/public/stylesheets/crashes.scss [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+		}
+
+		&--neutral {
+			color: map.get(c.$colors, "cool-gray-50");
+
+			i {
+				background-color: map.get(c.$colors, "cool-gray-20");
+			}
+		}
+	}
```
Comment (can-keklik) [inline plugins/crashes/frontend/public/stylesheets/crashes.scss:48]: We should convert this to a common util @itsiprikshit

File: plugins/crashes/frontend/public/javascripts/countly.views.js [hunk 5]
```diff
@@ -1,3756 +1,435 @@
...
+        </div>\
+        <div class="text-medium color-cool-gray-100 bu-columns bu-is-gapless bu-is-mobile">\
+        <h2>{{data && data.total}}</h2>\
+        <div :class="\'text-medium font-weight-bold crashes-trend crashes-trend--\' + colorClass">\
+        <i class="ion" :class="\'ion-\' + iconClass"></i>\
+        {{data && data.change}}\
+        </div>\
+        </div>\
+        </div>'
+    });
```
Comment (can-keklik) [inline plugins/crashes/frontend/public/javascripts/countly.views.js:45]: You could export this to a file :)

File: plugins/crashes/frontend/public/javascripts/countly.views.js [hunk 6]
```diff
@@ -1,3756 +1,435 @@
...
+    var CrashgroupView = countlyVue.views.BaseView.extend({
+        template: "#crashes-crashgroup",
+        components: {"crash-stacktrace": CrashStacktraceView},
+        data: function() {
+            return {
+                appId: countlyCommon.ACTIVE_APP_ID,
+                authToken: countlyGlobal.auth_token,
+                chartBy: "os_version",
+                diagnosticsOrder: ["ram", "disk", "battery", "running", "sessions"],
+                eventLogsEnabled: countlyGlobal.plugins.includes("crashes-event-logs"),
```
Comment (can-keklik) [inline plugins/crashes/frontend/public/javascripts/countly.views.js:156]: IE11 has no "includes" support :/

File: plugins/crashes/frontend/public/javascripts/countly.views.js [hunk 7]
```diff
@@ -1,3756 +1,435 @@
...
-                            if (!data) {
-                                CountlyHelpers.alert(jQuery.i18n.map["crashes.try-later"], "red");
-                            }
-                            else {
-                                self.resetSelection(true);
-                            }
+            },
+            stopEditingComment: function() {
+                this.$data.commentInput = "";
+                this.$data.commentBeingEdited = undefined;
```
Comment (can-keklik) [inline plugins/crashes/frontend/public/javascripts/countly.views.js:205]: Why not `this.commentInput` and instead `this.$data.commentInput`?

**Review by can-keklik — COMMENTED** [review]
There are reoccurring stuff, I tried to highlight an example for each kind.

File: plugins/crashes/package.json (hunk 1 above)
Comment (LVMBDV) [reply plugins/crashes/package.json:3]: This was for debugging purposes, it'll be removed before it hits master 😇

File: plugins/crashes/frontend/public/javascripts/countly.models.js (hunk 3 above)
Comment (LVMBDV) [reply plugins/crashes/frontend/public/javascripts/countly.models.js:208]: Anything to save some keystrokes :)

File: plugins/crashes/frontend/public/javascripts/countly.views.js (hunk 6 above)
Comment (LVMBDV) [reply plugins/crashes/frontend/public/javascripts/countly.views.js:156]: We should polyfill it, "indexof !== -1" is ugly 🥺

File: plugins/crashes/frontend/public/javascripts/countly.views.js (hunk 7 above)
Comment (LVMBDV) [reply plugins/crashes/frontend/public/javascripts/countly.views.js:205]: I guess it looked more consistent near `this.$otherStuff`. Is there a semantical difference?

**Pull request merged**